### DISCUSSION TITLE
Custom interface does not appear on UI when 'group'='relational'

### DISCUSSION THREAD

**Discussion by jsx101** [directus/directus#16322] [Archive]
I'm creating a custom interface with a many-to-one type. But when I set the "group" field to "relational" in the interface options _index.ts_ file, my custom interface does not show up in my Directus project UI.

The Directus version I'm using is 9.18.1

```
import { defineInterface } from '@directus/shared/utils';
import InterfaceComponent from './interface.vue';

export default defineInterface({
	id: 'jsxInterfaceProtoOne',
	name: 'Interface Prototype 1.9',
	icon: 'arrow_right_alt',
	component: InterfaceComponent,
	types: ['uuid', 'string', 'text', 'integer', 'bigInteger', 'alias'],
	relational: true,
	localTypes: ['m2o'],
	group: 'relational',
	options: ({ relations }) => {
		const collection = relations.m2o?.related_collection;

		return [
			{
				field: 'fieldsToDisplay',
				name: 'Display Fields',
				meta: {
					interface: 'system-fields',
					options: {
						collectionName: collection,
					},
				},
			},
			{
				field: 'template',
				name: '$t:interfaces.select-dropdown-m2o.display_template',
				meta: {
					interface: 'system-display-template',
					options: {
						collectionName: collection,
					},
				},
			},
			{
				field: 'enableCreate',
				name: '$t:creating_items',
				schema: {
					default_value: true,
				},
				meta: {
					interface: 'boolean',
					options: {
						label: '$t:enable_create_button',
					},
					width: 'half',
				},
			},
			{
				field: 'enableSelect',
				name: '$t:selecting_items',
				schema: {
					default_value: true,
				},
				meta: {
					interface: 'boolean',
					options: {
						label: '$t:enable_select_button',
					},
					width: 'half',
				},
			},
		];
	},
	recommendedDisplays: ['related-values']
});

```

I've tried it with "group" set to "standard" with everything else in _index.ts_ is the same and the interface is there. "Interface Prototype Zero" is another custom interface I made, it's "Interface Prototype  1.9" that doesn't appear.

### "group": "relational"
![image](https://user-images.githubusercontent.com/40978303/199966566-102a4768-e4a0-4ce5-b2db-cbfaa3d784c9.png)

### "group": "standard"
![image](https://user-images.githubusercontent.com/40978303/199967076-9600ac75-3d7b-4ae7-b2e4-72d7d7cdfc5f.png)

I've also tried removing the "group" field and used "groups" set to "M2O" following the documentation [here](https://docs.directus.io/extensions/interfaces.html#extension-entrypoint). I get the same result as setting "group" to "relational".

What configurations did I get wrong?

**Comment by azrikahar** [comment]
>  But when I set the "group" field to "relational" in the interface options index.ts file, my custom interface does not show up in my Directus project UI.

Based on _both_ of your screenshots above, could it be you were looking for it in `Text & Numbers` group, but it's actually (correctly) displayed in `Relational` group?

`Interface Prototype 1.9` should be under `Relational`, not `Text & Numbers` 👍

I've also confirmed that `group: 'relational'` worked for me:

https://user-images.githubusercontent.com/42867097/200267837-a2a84538-2c6b-4f3e-90b5-d512a4f2f049.mp4

> I've also tried removing the "group" field and used "groups" set to "M2O" following the documentation [here](https://docs.directus.io/extensions/interfaces.html#extension-entrypoint).

I believe the plural `groups` in the docs is outdated, so `group` should be the current and correct property. Ref https://github.com/directus/directus/discussions/13753#discussioncomment-3258731

**Reply by jsx101** [reply]
Yes, I see it now. Could've sworn I checked there too before but might have missed it. Thank you so much!
